README.md

```diff
@@ -47,12 +47,12 @@ g.AddTsigToDomain("example.com", tsig.UUID)
 for _, host := range []string{"198.51.100.1", "2001:DB8::1"} {
     g.AddSlaveToDomain("example.com", host)
 }
-// Step 8: Create a TSIG key
+// Create a TSIG key
 g.CreateTsig()
-//Step 9: Add the TSIG key to the domain
+// Add the TSIG key to the domain
 g.AddTsigToDomain("example.com", "85e7b6e3-4553-479b-b968-cd0c77143802")
-//Step 10: Get the nameservers which provide axfr services for that TSIG key in the BIND config example
-g.GetTsigBind("85e7b6e3-4553-479b-b968-cd0c77143802")
+// Get the nameservers which provide axfr services for that TSIG key in the BIND config example
+g.GetTsigBIND("85e7b6e3-4553-479b-b968-cd0c77143802")
```

Comment:
Sorry, I am mainly focused on my work, I overlooked that ; once again, I could have wrote this comment before...

CreateTsig and AddTsigToDomain are already there (lines 43 and 45), so it looks like you should only add the GetTsigBIND line (and its comment).